staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers.go

```diff
@@ -109,3 +109,11 @@ func CRDRemoveFinalizer(crd *CustomResourceDefinition, needle string) {
 	}
 	crd.Finalizers = newFinalizers
 }
+
+func Float64Ptr(f float64) *float64 {
```

Comment:
Are these only used in validation_test.go? Can we move them there and make them private?